src/components/mypage/SelectBuyerModal/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { IconButton, Divider, Avatar } from '@offer-ui/react'
+import type { ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type { SelectBuyerModalProps } from './types'
+import { toLocaleCurrency } from '@utils'
+
+export const SelectBuyerModal = ({
+  productName,
+  buyers,
+  handleClickReviewButton,
+  ...props
+}: SelectBuyerModalProps): ReactElement => {
+  const [selectedBuyerId, setSelectedBuyerId] = useState<number>(0)
```

Comment:
```suggestion
  const [selectedBuyerId, setSelectedBuyerId] = useState(0)
```
명시적으로 추론되는 값은 타입을 적지 않아도 괜찮을 것 같은데 어떠신가요?

Reply:
[적용](https://github.com/price-offer/offer-fe/pull/107/commits/dbe833d84018b54c74b18926d7f84e1c03b5ba2d) 완룡